### PR TITLE
Update Windows Grouping

### PR DESCRIPTION
**Pull request by Elitesparkle**
Add a gap when grouping windows horizontally.

The value is customizable by the user.

### CODE REVIEW

File: classes/class_instance.lua [hunk 1]
```diff
@@ -32,6 +32,8 @@ local modo_all = Details._detalhes_props["MODO_ALL"]
 local atributos = Details.atributos
 local sub_atributos = Details.sub_atributos
 
+local grouped_windows_horizontal_gap = 4
```
Comment (Flamanis) [inline classes/class_instance.lua:35]: This is bad. Local that can't be modified in any way and will severely disrupt user preferences

**Review by Flamanis — CHANGES_REQUESTED** [review]

**Comment by Flamanis** [comment]
Also, i question the need for such a feature in the first place. Adding a gap seems to be the opposite of what most want

File: classes/class_instance.lua (hunk 1 above)
Comment (Elitesparkle) [reply classes/class_instance.lua:35]: I know, it's incomplete.

**Comment by Elitesparkle** [comment]
Depending on the Skin being used, and especially for Custom Skins, the user may prefer having a small gap between windows that are grouped horizontally. While it is already possible to create such a gap by disabling grouping and manually aligning windows, using this method is worse than having an option for it because, due to the windows not being grouped, any change made to a window will have to be manually made to the other one.

**Comment by Tercioo** [comment]
Merging, thank you!

**Pull request merged**